plugins/wasi_llm/wasillmfunc.cpp

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2024 Second State INC
+
+#include "wasillmfunc.h"
+#include "common/spdlog.h"
+
+#include <string>
+#include <string_view>
+
+namespace WasmEdge {
+namespace Host {
+
+Expect<WASILLM::ErrNo>
+WasiLLMModelCreate::bodyImpl(const Runtime::CallingFrame &Frame,
+                             uint32_t CheckPointPath,
+                             uint32_t CheckPointPathLen) {
+  (void)Frame;
```

Comment:
Use `maybe_unused` instead.
https://en.cppreference.com/w/cpp/language/attributes/maybe_unused

Reply:
It's inappropriate to use `[[maybe_unused]]` attribute here, since what I really want to do is simply suppress the compiler warnings temporarily, they are all **used**. Considering we're gonna remove these very soon, using the old C style `(void)x` is good enough. What's more, using `[[maybe_unused]]` to suppress the warning needs to modify the declaration of the function, which makes the diff harder to read when we change it later.
```c++
WasiLLMModelCreate::bodyImpl([maybe_unused]] const Runtime::CallingFrame &Frame,
                             [maybe_unused]] uint32_t CheckPointPath,
                             [maybe_unused]] uint32_t CheckPointPathLen) {
```